tree/dataframe/test/dataframe_cache.py

```diff
@@ -1,36 +1,46 @@
 import unittest
 import ROOT
+import os
 
 RDataFrame = ROOT.ROOT.RDataFrame
 
 class Cache(unittest.TestCase):
     @classmethod
     def setUp(cls):
-      code = ''' {
-      const char* treeName = "t";
-      const char* fileName = "fileName.root";
-      TFile f(fileName, "RECREATE");
-      TTree t(treeName, treeName);
-      float arr[4];
-      t.Branch("arr", arr, "arr[4]/F");
-      for (auto i : ROOT::TSeqU(4)) {
-         for (auto j : ROOT::TSeqU(4)) {
-            arr[j] = i + j;
-         }
-         t.Fill();
-      }
-      t.Write();
-      }'''
-      ROOT.gInterpreter.Calc(code)
+        # Necessary because old and new Cppyy use a different syntax for
```

Comment:
New cppyy still allows the round bracket syntax in some cases. But in this case it has no means to know that you want an explicit instantiation since you are passing a string - and it could be that you are just trying to make a call with a string parameter (which e.g. `Take` accepts)
See for instance an analogous case:
https://github.com/root-project/roottest/blob/master/python/cpp/PyROOT_advancedtests.py#L256